hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegionServer.java

```diff
@@ -1819,7 +1785,7 @@ protected void chore() {
         if (r.shouldFlush(whyFlush)) {
           FlushRequester requester = server.getFlushRequester();
           if (requester != null) {
-            long randomDelay = (long) RandomUtils.nextInt(0, rangeOfDelay) + MIN_DELAY_TIME;
+            long randomDelay = RandomUtils.nextLong(0, rangeOfDelayMs) + MIN_DELAY_TIME;
```

Comment:
Will this change the possible delays? Wonder if nextInt was intentionally chosen?

Reply:
No I don't believe it will. I think `int` was used down here because the value is read out as an `int` using `conf.getInt()` -- which it still uses. The value will remain as whatever was read from the config. I'm simply converting the value to a `long` slightly earlier.